benchmarks/log_likelihood.hpp

```diff
@@ -4,8 +4,12 @@ namespace xt
 	{
 		auto s = pow((data - mean), 2) / (2 * (sigma * sigma));
 		tensor<double, 1> pdfs = exp(-s);
+#if 0
 		pdfs /= std::sqrt(2 * numeric_constants<double>::PI) * sigma;
```

Comment:
Does this not compile anymore?

Reply:
Yes, see https://github.com/QuantStack/xtensor-python/issues/203